src/util/selectors.ts

```diff
@@ -129,3 +129,14 @@ export function selectKanTildeleVeileder(
     if (!oppfolging || !tilgangTilBrukersKontor) return false;
     return oppfolging.underOppfolging && tilgangTilBrukersKontor.tilgangTilBrukersKontor;
 }
+
+export function selectTelefonnummer(personalia: Personalia | undefined): StringOrNothing {
+    if (!personalia?.telefon) return '';
+    if (personalia.telefon.find(entry => entry.prioritet === '1'))
+        return personalia.telefon.find(entry => entry.prioritet === '1')?.telefonNr;
+    if (personalia.telefon.find(entry => entry.prioritet === '2'))
+        return personalia.telefon.find(entry => entry.prioritet === '2')?.telefonNr;
+    if (personalia.telefon.find(entry => entry.prioritet === '3'))
+        return personalia.telefon.find(entry => entry.prioritet === '3')?.telefonNr;
+    return;
+}
```

Comment:
```suggestion
export function selectTelefonnummer(personalia: Personalia | undefined): StringOrNothing {
    const telefonEntries = personalia?.telefon ?? [];
    const telefonPriorities = ['1', '2', '3'];
    for (const priority of telefonPriorities) {
      const entry = telefonEntries.find((entry) => entry.prioritet === priority);
      if (entry) {
        return entry.telefonNr;
      }
    }
    return undefined;
  }
```
Hva med noe sånt no? Delvis refaktorert av co-pilot, med små justeringer 🤓 Da unngår man å kjøre `.find()` så mange ganger! Ta gjerne og test om det funker som tenkt

Reply:
Eller no slikt? da slepper man å traversere om det ikke finnes noen 

export function selectTelefonnummer(personalia: Personalia | undefined): StringOrNothing {
    const telefonEntries = personalia?.telefon ?? [];
    return telefonEntries.filter(tlfEntry => tlfEntry.telefonNr).sort((a, b) => Number(a.prioritet) - Number(b.prioritet))[0]?.telefonNr
  }

Reply:
Fancy 😁 Med en kort beskrivende kommentar så er det nice!